### DISCUSSION TITLE
ModuleNotFoundError: No module named 'manim_imports_ext' when simulating ManimGL

### DISCUSSION THREAD

**Discussion by davidjun-vet** [3b1b/manim#1859] [Q&A]
I'm trying to simulate the  epidemic models of 3b1b  that he made in 2020. I copied the codes in _2020/sir.py . However when I  write "manimgl project1.py IntroducePlot -pql" on cmd, it says "ModuleNotFoundError: No module named 'manim_imports_ext'"
what can I do? Please help me figure this out. Thank you a lot.

**Comment by davidjun-vet** [comment]
actually I used the code _2020/covid.py

**Comment by GuanZhuiGuo** [comment]
me too
